### DISCUSSION TITLE
Proposal for Duplicate Issue Management

### DISCUSSION THREAD

**Discussion by sayyid5416** [fast4x/RiMusic#1602] [General]
Given the rapid growth of the app, it's fantastic to see more users engaging on GitHub by reporting issues or suggesting features they'd like to see. However, one challenge we're facing is the duplication of entries. Many users aren't taking the time to check if their idea or bug report has already been addressed or logged, resulting in unnecessary redundancy.

To address this issue, I propose users, repository maintainers, or anyone spotting a duplicate issue should leave a comment indicating that it's a duplicate of `#<num>`. Subsequently, the maintainer can close the duplicate, consolidating the discussion onto one thread. This approach streamlines issue tracking process, ensuring better organization and making it easier to address user feedback effectively.

**Comment by AnyoneMouse** [comment]
I suggested this: https://github.com/fast4x/RiMusic/discussions/92#discussioncomment-8577835 2 months ago. But maybe fast4x didn't notice or maybe he just prefers everyone to have freedom. I wonder what's the actual case though. 🤔

**Reply by sayyid5416** [reply]
I agree, let's see if fast4x has anything to say about it. It's for the betterment of the repository, but maybe he has something else in mind 😅

**Reply by AnyoneMouse** [reply]
> I agree, let's see if fast4x has anything to say about it. It's for the betterment of the repository, but maybe he has something else in mind 😅

Yeah. I have a feeling he prefers everyone to have freedom to share ideas. Because he thinks if someone tries to suggest an idea and sees that has already been suggested they'll (maybe) feel kinda sad that they weren't the first to think/suggest it. 

Okay nevermind I'm speculating too many things. I'll stop before I stop making sense. 💀💀

**Reply by fast4x** [reply]
That’s right, I like to think that everyone should feel free to write their own thoughts without thinking that someone is doing it because they arrived later.

The great thing that I really like is that you put the reference to the duplicate ticket. I love your behavior because it makes it easier for me to research when I develop that feature as well as recognize you as a ticket manager.

Thanks you 🎉🤗
